src/lib/Game.cpp

```diff
@@ -131,3 +201,9 @@ void Game::spawnEnemies(size_t count){
 
 
 
+void Game::endGame(){
+    std::cout << "Ended game!\n";
```

Comment:
would this function where we make our game end page? or would it call a seperate function